components/ruby/ruby-32/Makefile

```diff
@@ -56,15 +57,19 @@ RUBY_VER= $(word 1,$(RUBY_VER_WORDS)).$(word 2,$(RUBY_VER_WORDS))
 RUBY_LIB_VER= $(RUBY_VER)
 
 PROTORUBYDIR= $(PROTO_DIR)/usr/ruby/$(RUBY_VER)
+
 ifeq ($(MACH), i386)
-GNU_CPU=x86_64
+RUBY_CPU=x86_64
 else
-GNU_CPU=sparc
+RUBY_CPU=sparc
 endif
-PROTO_RBCONFIG_FILE= $(PROTORUBYDIR)/lib/ruby/$(RUBY_LIB_VER)/$(GNU_CPU)-$(GNU_OS)/rbconfig.rb
+
+PROTO_RBCONFIG_FILE= $(PROTORUBYDIR)/lib/ruby/$(RUBY_LIB_VER)/$(RUBY_CPU)-$(GNU_OS)/rbconfig.rb
 
 # these macros are used in the package manifest
-PKG_MACROS += RUBY_VER=$(RUBY_VER) RUBY_LIB_VER=$(RUBY_LIB_VER) GNU_CPU=$(GNU_CPU) GNU_OS=$(GNU_OS)
+PKG_MACROS += RUBY_VER=$(RUBY_VER) RUBY_LIB_VER=$(RUBY_LIB_VER) RUBY_CPU=$(RUBY_CPU) GNU_OS=$(GNU_OS)
```

Comment:
I suggest to split this out to several lines (one per macro).  Are `RUBY_LIB_VER` and `GNU_OS` really needed here?

Reply:
I removed GNU_OS, you are right, it is not needed anymore. But I won't touch RUBY_LIB_VER, this was originally done by Friedrich and for me it looks like it is still needed because of the man-page stuff he's doing with it.

Reply:
Is the _stuff_ done in the manifest?

Reply:
Yepp:
pkg://userland/runtime/ruby-32@3.2.6,5.11-2024.0.0.1:20241226T151118Z
PUBLISHED

Reply:
> Yepp: pkg://userland/runtime/ruby-32@3.2.6,5.11-2024.0.0.1:20241226T151118Z PUBLISHED

How this proves it is used?

Reply:
see here:
https://github.com/OpenIndiana/oi-userland/blob/dc4e8a1d14d74d643de2fecad12a3c025084f796/components/ruby/ruby-32/manifests/sample-manifest.p5m
and here:
https://github.com/OpenIndiana/oi-userland/blob/dc4e8a1d14d74d643de2fecad12a3c025084f796/components/ruby/ruby-32/manifests/sample-manifest.p5m

Reply:
> see here: https://github.com/OpenIndiana/oi-userland/blob/dc4e8a1d14d74d643de2fecad12a3c025084f796/components/ruby/ruby-32/manifests/sample-manifest.p5m and here: https://github.com/OpenIndiana/oi-userland/blob/dc4e8a1d14d74d643de2fecad12a3c025084f796/components/ruby/ruby-32/manifests/sample-manifest.p5m

What exactly should I look for there?